.github/workflows/release.yml

```diff
@@ -39,6 +37,9 @@ jobs:
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
```

Comment:
> Use tonistiigi/binfmt for better emulation

I'm still learning- How is it better than the default? 🤔

Reply:
Those are qemu static binaries, I don't know exact performance benefit but seen @crazy-max to use it everywhere

Reply:
> I'm still learning- How is it better than the default? 🤔

This image is already the default one: https://github.com/docker/setup-qemu-action#inputs